docs/contributing/contributing.md

```
@@ -5,11 +5,72 @@ description: Guide on how to contribute to this project
 
 ## General Feedback
 
 If you have a general feedback about our project, please do not open an issue but instead please fill in this [form](https://forms.gle/LRUq3vsFnE1QCLiA6)
 
+## Fork & branch processing
+
+First fork the origin repository to your own github repository, then clone the repository to your local computer.
+
+```bash
+git clone https://github.com/Your Account name/clima.git
+cd clima
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output

(MD014, commands-show-output)